VideoWeb/VideoWeb.Services.Generator/video-api.nswag

```diff
@@ -3,7 +3,7 @@
   "defaultVariables": null,
   "documentGenerator": {
     "fromDocument": {
-        "url": "https://vh-video-api-dev.azurewebsites.net/swagger/v1/swagger.json",
```

Comment:
Lionel can you change this back? It better to generate swagger from dev not local

Reply:
reverted to point to dev

Reply:
cheers